gradle/project.gradle.kts

```diff
@@ -104,31 +96,34 @@ dependencies {
         implementation(kotlin("stdlib-jdk8"))
         implementation(kotlin("reflect"))
         implementation("org.jetbrains.kotlinx", "kotlinx-coroutines-core")
-        implementation("org.jetbrains.kotlinx", "kotlinx-coroutines-javafx", "1.0.1")
+        implementation("org.jetbrains.kotlinx", "kotlinx-coroutines-javafx", "1.1.0")
+    }
+
+    "Logging" requires {
+        implementation("org.slf4j", "slf4j-api")
+        implementation("org.slf4j", "jul-to-slf4j", "1.7.25")
+        implementation("ch.qos.logback", "logback-classic", "1.2.3")
+        implementation("io.github.microutils", "kotlin-logging", "1.6.22")
     }
 
     "Audio" requires {
-        implementation("uk.co.caprica", "vlcj", "3.10.1")
+        implementation("uk.co.caprica", "vlcj", "3.11.0")
         implementation("de.u-mass", "lastfm-java", "0.1.2")
         implementation("com.github.goxr3plus", "Jaudiotagger", "V2.2.6")
     }
 
     "JavaFX" requires {
         implementation("de.jensd", "fontawesomefx", "8.9")
         implementation("org.reactfx", "reactfx", "2.0-M5")
-        implementation("eu.hansolo", "tilesfx", "1.6.4") { exclude("junit", "junit") }
```

Comment:
I'ts ok to update things like logging or vlcj, because we rely on the their functionality, not api. It is ok to contiuously update language because of features and bugfixes. But libraries are not to be updated so liberally. Even if the API does not change, there may be substantial changes under hood that require making sure the update is safe. Because of that we update when we need to not when we can. So pls excersize caution and restraint with libs where the dependency is too strong.

Reply:
I was hoping that he had somehow removed junit from the dependency graph in the new version

Reply:
i see